data/en/pi.json

```diff
@@ -4,25 +4,34 @@
 	"syntax":"pi()",
 	"returns":"numeric",
 	"related":[],
-	"description":" Gets the mathematical constant p, accurate to 15 digits.\n The number 3.14159265358979.",
-	"params": [
-	],
+	"description":" Gets the mathematical constant p, accurate to 11 digits.\n The number 3.14159265358979.",
```

Comment:
Hmm, I think it still returns the 15 digits but when you output it, you only see 11, right?

Reply:
Yes, but why?